themes/openy_themes/openy_rose/templates/page/page.html.twig

```diff
@@ -73,7 +73,7 @@
       <div class="search-form-wrapper col-xs-12">
         <form method="get" action="{{ base_path }}search">
```

Comment:
We can use Drupal path in the action instead of hardcoded alias. Since we have a landing page NID in the config now. So the form will work even if the alias is changed in the landing.

Reply:
Let's make it after rest of PR merged

Reply:
Ok, they have been merged, so we can do it now.

Reply:
Done